lib/cli/commands/build.js

```diff
@@ -37,6 +38,38 @@ build.builder = function(cli) {
 			default: false,
 			type: "boolean"
 		})
+		.option("include-dependency", {
+			describe: "A list of dependencies to be included into the build process; using the '*' as value is an" +
```

Comment:
It doesn't change the information content which separator you choose. Separate sentences are perhaps a bit easier to read.

Reply:
I used semicolons because the other descriptions are more in bullet point style, not in complete sentences. I've now changed to separate sentences to improve the readability